README.md

```diff
@@ -201,6 +201,12 @@ For the VoxPopuli ASR task, we provide Transformer baselines, fine-tuned wav2vec
 
 Please refer to the [S2T examples](https://github.com/pytorch/fairseq/tree/master/examples/speech_to_text) for the use of baseline model checkpoints.
 
+## Wav2letter C++ implementation
```

Comment:
Maybe we can move this note to the end of Section `#  Pre-trained Models --- ## wav2vec 2.0` (After the sentence that cites Chaitanya's paper)?